runtime/doc/vim_diff.txt

```diff
@@ -541,6 +541,8 @@ Commands:
 - |:wincmd| accepts a count.
 - `:write!` does not show a prompt if the file was updated externally.
 - |:=| does not accept |ex-flags|. With an arg it is equivalent to |:lua=|
+- `:set {option}<` removes local value for all |global-local| options.
+- `:setlocal {option}<` copies global value to local value for all options.
```

Comment:
in vim_diff, these can live in the Options section of nvim-features. in fact there is already a related note on line 342.

the nvim-changed section should rarely be used.